frontend/api/boardService.tsx

```diff
@@ -1,22 +1,36 @@
 import fetchData from "../utils/fetchData";
-import { BoardIdTypeWithToken, BoardType, BoardTypeWithToken } from "../types/board";
+import {
+  BoardIdTypeWithToken,
+  BoardType,
+  BoardTypeWithToken,
+  UpdateTitleWithToken,
+} from "../types/board";
 import { Nullable } from "../types/common";
 
 export const postBoard = ({ newBoard, token }: BoardTypeWithToken): Promise<BoardType> => {
   return fetchData(`/boards`, "POST", JSON.stringify(newBoard), token);
 };
 
-export const putBoard = ({ newBoard, token }: BoardTypeWithToken): Promise<BoardType> => {
-  return fetchData(`/boards/${newBoard._id}`, "PUT", JSON.stringify(newBoard), token);
+export const updateTitleBoard = ({
```

Comment:
Nitpicking, just to improve readibility

```suggestion
export const updateBoardTitle = ({
```

Don't forget to change in other places